src/v0/destinations/mp/transform.js

```diff
@@ -331,7 +331,7 @@ const processGroupEvents = (message, type, destination) => {
           $token: destination.Config.token,
           $distinct_id: message.userId || message.anonymousId,
           $set: {
-            [groupKey]: [groupKeyVal]
+            [groupKey]: groupKeyVal
```

Comment:
Have we done the impact analysis ?

Reply:
Yes, actually suppose for `groupKey` `company` if value `RudderStack` was passed it was showing up as `[RudderStack]` in mix panel dashboard which should not be the case. Ref:  [here](https://rudderlabs.slack.com/archives/C03ESKCMASF/p1673057546171749)